internal/core/src/exec/expression/BinaryArithOpEvalRangeExpr.cpp

```diff
@@ -0,0 +1,748 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "BinaryArithOpEvalRangeExpr.h"
+
+namespace milvus {
+namespace exec {
+
+void
+PhyBinaryArithOpEvalRangeExpr::Eval(EvalCtx& context, VectorPtr& result) {
```

Comment:
we should all convert this expression to binary range

Reply:
yes. but now can not find a solution to convert it quickly, maybe next issue to solve it